src/main/java/com/prgrms/be/jpa/domain/Customer.java

```diff
@@ -0,0 +1,36 @@
+package com.prgrms.be.jpa.domain;
+
+import lombok.AllArgsConstructor;
+import lombok.Getter;
+import lombok.NoArgsConstructor;
+
+import javax.persistence.Entity;
+import javax.persistence.Id;
+import javax.persistence.Table;
+import javax.validation.constraints.NotBlank;
+import javax.validation.constraints.NotNull;
+import javax.validation.constraints.Size;
+
+@Getter
+@AllArgsConstructor
+@NoArgsConstructor
+@Entity
+@Table
+public class Customer {
+
+    @Id
+    @NotNull
+    private Long id;
+    @Size(min = 1, max = 5, message = "이름을 1 ~ 5자 이내로 입력해주세요.")
```

Comment:
```suggestion
    private Long id;
    
    @Size(min = 1, max = 5, message = "이름을 1 ~ 5자 이내로 입력해주세요.")
```
별건 아니지만, 통일성을 맞추면 좋겠어요.